test/Garnet.test/RespCustomCommandTests.cs

```diff
@@ -10,13 +10,62 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using Garnet.common;
 using Garnet.server;
 using NUnit.Framework;
 using NUnit.Framework.Legacy;
 using StackExchange.Redis;
+using Tsavorite.core;
 
 namespace Garnet.test
 {
+    public class LargeGet : CustomProcedure
+    {
+        public override bool Execute(IGarnetApi garnetApi, ArgSlice input, ref MemoryResult<byte> output)
+        {
+            int offset = 0;
+            var key = GetNextArg(input, ref offset);
+            for (int i = 0; i < 120_000; i++)
+            {
+                garnetApi.GET(key, out var outval);
+                garnetApi.FreeBuffer(ref outval);
```

Comment:
seems wasteful to need to call for every single item. can they directly get the old status and rewind to that?

Reply:
Added APIs to retrieve offsets and reset to any specific offset in the past. 
The test will now avoid invoking this every time and do it only periodically to avoid expanding the buffer too much.